modules/core/perf/perf_lut.cpp

```diff
@@ -24,4 +24,42 @@ PERF_TEST_P( SizePrm, LUT,
     SANITY_CHECK(dst, 0.1);
 }
 
+PERF_TEST_P( SizePrm, LUT_multi,
+             testing::Values(szQVGA, szVGA, sz1080p)
+           )
+{
+    Size sz = GetParam();
+
+    int maxValue = 255;
+
+    Mat src(sz, CV_8UC3);
+    randu(src, 0, maxValue);
+    Mat lut(1, 256, CV_8UC1);
+    randu(lut, 0, maxValue);
+    Mat dst(sz, CV_8UC3);
+
+    TEST_CYCLE() LUT(src, lut, dst);
+
+    SANITY_CHECK_NOTHING();
+}
+
+PERF_TEST_P( SizePrm, LUT_multi2,
```

Comment:
My thoughts on dropping this test:

1. This patch does not bring speedup in case of multiple channels lut.
2. It does not check sanity.
3. You have added a regression test to test the accuracy of multi-channel lut.